packages/connection/tests/php/test_Manager.php

```diff
@@ -49,6 +52,32 @@ public function test_is_active_when_not_connected() {
 		$this->assertFalse( $this->manager->is_active() );
 	}
 
+	public function test_api_url_defaults() {
```

Comment:
Would be great to add some more tests (and remove tests that no longer make sense to be in the default Jetpack suite), but that's definitely not a requirement for this PR; we can do it in another PR.